src/headfirst/designpatterns/factory/pizzas/Pizza.java

```diff
@@ -1,12 +1,13 @@
 package headfirst.designpatterns.factory.pizzas;
 
 import java.util.ArrayList;
+import java.util.List;
 
 abstract public class Pizza {
 	String name;
 	String dough;
 	String sauce;
-	ArrayList<String> toppings = new ArrayList<String>();
+	List<String> toppings = new ArrayList<String>();
```

Comment:
You can use a diamond operator:
 `List<String> toppings = new ArrayList<>();`

Reply:
You are right @vaambival. The reason I didn't do this is because this may not be compatible with older Java versions.